hikari/impl/rest.py

```diff
@@ -473,6 +514,14 @@ async def close(self) -> None:
         # https://github.com/aio-libs/aiohttp/issues/1925#issuecomment-715977247
         await asyncio.sleep(0.25)
 
+    def _get_live_attributes(self) -> _LiveAttributes:
+        if not self._live_attributes:
+            self._live_attributes = _LiveAttributes.build(
+                self._max_rate_limit, self._http_settings, self._proxy_settings
+            )
+
+        return self._live_attributes
```

Comment:
Why cant this be in the `__init__`?

Reply:
it builds objects which can only be built when within a live event loop (ClientSession, Event, RESTBucketManager and ManualRatelimiter)